templates/home.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="col my-auto">
+  <h1>URL Shortener</h1>
+  <form id="url-form" action="/" method="POST">
+    <input type="text" class="form-control"  id="url" name="url" placeholder="Enter url... (include http:// at the beginning)" />
+    <input class="form-control mt-1"  type="submit" id="url-submit" value="Submit URL">
+  </form>
+  {% if url %}
+    <div class="short-url">
+      <p>Here is your generated URL:</p>
+      <a href="/{{url}}">{{url}}</a>
```

Comment:
For users, it's really useful that we could click on the generated URL directly

Reply:
😄